iceoryx_hoofs/test/moduletests/test_cxx_variant.cpp

```diff
@@ -197,14 +197,7 @@ TEST_F(variant_Test, EmplaceInvalidElementCompileTimeCheck)
 {
     ::testing::Test::RecordProperty("TEST_ID", "6fa3b290-8249-4825-8eac-72235a06710e");
     GTEST_SKIP() << "Compile time check, if you uncomment this test, the compiler will fail";
-    // EXPECT_THAT(sut.emplace< unsigned int >(0), Eq(false));
-}
-
-TEST_F(variant_Test, EmplaceWhenAlreadyDifferentTypeAssignedDoesNotWork)
-{
-    ::testing::Test::RecordProperty("TEST_ID", "f2c0825a-349c-4acb-9643-cb046f7e0310");
-    sut.emplace<int>(123);
-    EXPECT_THAT(sut.emplace<float>(123.F), Eq(false));
+    // sut.emplace<unsigned int>(0);
```

Comment:
Not sure it is useful to have this as there is no automatic checking that this does not compile. Nothing for this PR though ...